Marlin/src/inc/SanityCheck.h

```diff
@@ -2201,6 +2201,13 @@ static_assert(hbm[Z_AXIS] >= 0, "HOMING_BUMP_MM.Z must be greater than or equal
   #endif
 #endif
 
+/**
+ * Make sure PASSWORD_FEATURE is not enabled with SD_AUTOOPEN_MENU
+ */
+#if ENABLED(SD_AUTOOPEN_MENU) && ENABLED(PASSWORD_FEATURE)
```

Comment:
Suggestion:
`BOTH(SD_AUTOOPEN_MENU, PASSWORD_FEATURE)`

Reply:
@qwewer0 is correct, although it is understandable to not know all the available macros 😉

Reply:
Thanks for the feedback! updated the SanityCheck.h just now

As for the CR10_STOCKDISPLAY, I'm not sure how to code an exception for similar LCDs, but it might work on the REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER or similar that I haven't tested